ports/sfgui/portfile.cmake

```
@@ -1,24 +1,26 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO TankOs/SFGUI
-    REF 0.4.0
-    SHA512 15456c6080b7095bcdcec08489b2b91b5cfc36cdf3c0b645b305072e7e835837eb4f95b59371ff176630b2b7ae51da475d8ea0bde5ff7fc0ba74c463bf5f54cf
+    REF 1.0.0
+    SHA512 cc543cd44cf7d922d086748eea57d75069682649aa5f788bfc6ec3baa7bf7f9a010b4314d1a1875648cfaabf8d9efef130843ac1848d1112b5d53fd508768e41
     HEAD_REF master
-    PATCHES
-        "001-fix-corefoundation-link.patch"
-	"002-add-limits-header.patch"
 )
 
-file(REMOVE "${SOURCE_PATH}/cmake/Modules/FindSFML.cmake")
-
-string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "dynamic" SFGUI_BUILD_SHARED_LIBS)
+if(VCPKG_LIBRARY_LINKAGE STREQUAL "dynamic")
+    set(SFGUI_BUILD_SHARED_LIBS ON)
+    set(SFML_STATIC_LIBRARIES OFF)
+else()
+    set(SFGUI_BUILD_SHARED_LIBS OFF)
+    set(SFML_STATIC_LIBRARIES ON)
+endif()
 
 vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}"
     OPTIONS
         -DSFGUI_BUILD_DOC=OFF
         -DSFGUI_BUILD_EXAMPLES=OFF
         -DSFGUI_BUILD_SHARED_LIBS=${SFGUI_BUILD_SHARED_LIBS}
+        -DSFML_STATIC_LIBRARIES=${SFML_STATIC_LIBRARIES}
```

... with `SFGUI` being the port name.
```suggestion
        -DSFML_STATIC_LIBRARIES=${SFGUI_BUILD_SHARED_LIBS}
```

FTR sfml is a dependency. At this point, you don't know if triplet customization chose a different linakge for sfml. So ideally `SFML_STATIC_LIBRARIES` wouldn't be set at all but simply use what is installed in the vcpkg tree.
```suggestion
```

---

I reapplied the patch in SFML, see [comment](https://github.com/microsoft/vcpkg/pull/42913#discussion_r1962992104), as such this is no longer needed.